helix-term/src/commands.rs

```diff
@@ -2253,14 +2254,21 @@ fn buffer_picker(cx: &mut Context) {
         path: doc.path().cloned(),
         is_modified: doc.is_modified(),
         is_current: doc.id() == current,
+        used_at: doc.used_at,
     };
 
+    let mut items = cx
+        .editor
+        .documents
+        .iter()
+        .map(|(_, doc)| new_meta(doc))
+        .collect::<Vec<BufferMeta>>();
+
+    // mru
+    items.sort_by(|a, b| b.used_at.cmp(&a.used_at));
```

Comment:
```suggestion
    // Sort so that most recently focused is on top
    items.sort_unstable_by_key(|item| std::cmp::Reverse(item.used_at));
```

In general I think sorting unstable is preferrable - it's unlikely that two items will have the same exact focused-at instant and even if they did, the ordering between them wouldn't be a big deal, right?

Reply:
done